pkg/actions/nodegroup/upgrade.go

```diff
@@ -34,10 +34,19 @@ func (m *Manager) Upgrade(options managed.UpgradeOptions) error {
 		return managedService.UpgradeNodeGroup(options)
 	}
 
-	return m.upgradeAndWait(options)
+	if err := m.upgrade(options); err != nil {
+		return err
+	}
+
+	if wait {
+		return m.waitForUpgrade(options)
+	}
+
+	return nil
```

Comment:
We should also log that the request has been submitted if `wait` is not set:

```suggestion
	logger.Info("nodegroup upgrade request submitted successfully")
	return nil
```

Reply:
@cPu1 Added the log info.